api/pom.xml

```diff
@@ -65,10 +61,10 @@
 		<dependency>
 			<groupId>org.codehaus.jackson</groupId>
 			<artifactId>jackson-core-asl</artifactId>
-		</dependency>		
+		</dependency>
 		<dependency>
 			<groupId>javax.servlet</groupId>
-			<artifactId>servlet-api</artifactId>
+			<artifactId>javax.servlet-api</artifactId>
```

Comment:
Can we eliminate the need for this dependency in the api module while we are at this?  Not really in the scope of this ticket, but generally the API should not have this kind of web dependency.